asf_heat_pump_suitability/pipeline/prepare_features/anchor_properties.py

```diff
@@ -103,12 +102,12 @@ def load_gdf_and_process_poi() -> gpd.GeoDataFrame:
     return anchor_properties
 
 
-def identify_anchor_properties_gdf() -> gpd.GeoDataFrame:
+def identify_anchor_properties_df() -> pl.DataFrame:
```

Comment:
I changed the output of this function to a `polars.DataFrame` object so that it can be joined to the EPC dataset (which is also a `polars.DataFrame`) in the `run_add_features.py` script.